src/instrument.cpp

```diff
@@ -918,26 +972,61 @@ Result BytecodeRewriter::rewriteClass() {
     const u8* cpool_end = _src;
     put(cpool_start, cpool_end - cpool_start);
 
+    if (_target_methods == nullptr) {
+        for (int i = 1; i < _cpool_len; i += _cpool[i]->slots()) {
+            if (_cpool[i]->tag() != JVM_CONSTANT_Class) continue;
+
+            u16 name_idx = _cpool[i]->info();
+            const auto& it = _targets->find(_cpool[name_idx]->as_string());
```

Comment:
This lookup logic differs from the lookup logic in `ClassFileLoadHook`.

Reply:
Fixed in 5696577a796f340d9ba2ce6ddd8a3f6bf12986c7